### PR TITLE
10 setup event entities

### PR DESCRIPTION
**Pull request by missionStrawberry**
Fixed the file organization (I believe) and wrote the Event and EventFactory files along with their corresponding test files (100% coverage). Used sql.Time to store the start and end time. I couldn't find other libraries that worked the way I wanted it to, but this one works.

### CODE REVIEW

**Comment by missionStrawberry** [comment]
Also take look at how my javadoc and file headers are formatted. We need to figure out our conventions for those.

**Review by DanielLivshits — APPROVED** [review]
The entities and tests look good, just needed to change the time class we were using (LocalTime should work better for our purposes)

File: src/main/java/entities/Event.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/**
+ * Event class.
+ *
+ * Author: Anna Myllyniemi
+ * Date created: Nov 9 2022
+ *
```
Comment (missionStrawberry) [inline src/main/java/entities/Event.java:6]: Add "Modified by: Daniel Livshits" in between line 4 and 5

**Comment by missionStrawberry** [comment]
Now that I'm thinking about it I'm not sure if my test methods have good naming.

**Review by Fard-Faru — APPROVED** [review]
Looks Good!

**Comment by Fard-Faru** [comment]
I'll review again after Anna's comments are resolved

**Comment by DanielLivshits** [comment]
Capitalization on test names inconsistent with the naming conventions outlined in lecture. Otherwise, looks good.

**Review by Fard-Faru — APPROVED** [review]
Looks Good!

**Review by eatingcorn — APPROVED** [review]
Everything looks great

**Review by DanielLivshits — APPROVED** [review]
Looks good!

**Pull request merged**